### DISCUSSION TITLE
przywroc-czytelnosc-stron-internetowych/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#119] [General]
# przywroc-czytelnosc-stron-internetowych/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/przywroc-czytelnosc-stron-internetowych/

**Comment by xpil** [comment]
Ironiczne, że artykuł o czytelności stron internetowych otwiera się w bardzo nieczytelny sposób na urządzeniach mobilnych :) A konkretnie, jeżeli smartfon jest w orientacji pionowej, strona otwiera się z prawym marginesem na połowę szerokości ekranu, czyli sam artykuł jest bardzo wąski i wysoki, i po lewej stronie. Oczywiście można zrobić zoom i wtedy jest w miarę OK; można też obrócić ekran do poziomu i wtedy jest całkiem OK. Na pececie otwiera się bez skazy i zmazy.

**Reply by idarek** [reply]
A możesz napisać coś więcej. Jakie urządzenie itp lub zamieścić zrzut ekranu.

Sprawdziłem na iPhone 13/13 pro max oraz OnePlus Nord i na obu urządzeniach mobilnych otwiera się prawidłowo.

**Reply by xpil** [reply]
Telefon to Samsung Galaxy S21 Ultra 5G (SM-998B/DS)
Android 14 (One UI 6.1)
Kernel 5.4.242 z groszami.
Załączam screenshoty z różnych przeglądarek. Te dwa poprawne są z Firefoxa (zwykły + Focus).

Reszta to:
- Telegram (natywny podgląd)
- Vivaldi
- Chrome

Czyli w skrócie: w Firefoxie działa, wszędzie indziej nie działa.

**Reply by idarek** [reply]
Skoro 2 pierwsze są poprawne, możesz przeninąć w dół i **zobaczyć który element to powoduje,** i zrobić temu screenshot.

Osobiście sprawdziłem w Chrome na iPhone i Androidzie i nie doświadczyłem tego problemu więc chciałbym to naprawić.

ps. chyba wiem który element, ale chaiłbym potwierdzić.

**Reply by idarek** [reply]
Dzięki, poparze.

ps. Vivaldi na iPhone oraz Androidzie działa poprawnie.

**Reply by idarek** [reply]
Heja, dzięki za zwrócenie uwagi. Znalazłem [co to powodowało](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag).

Poprawiłem :)

Ah ta wsteczna kompatybilność.

**Reply by xpil** [reply]
Teraz działa jak trzeba, szacun!

**Comment by spexpl** [comment]
Firefox ma taki czytnik wbudowany i jest dostępny pod F9. Do tego mam wrażenie, iż AdGuard jednak sobie lepiej teraz radzi z reklamami niż AdBlock.
